Services/User/classes/class.ilUserXMLWriter.php

```diff
@@ -252,7 +252,7 @@ private function __handlePreferences(array $prefs, array $row): void // Missing
         $prefs["mail_incoming_type"] = $mailOptions->getIncomingType();
         $prefs["mail_address_option"] = $mailOptions->getEmailAddressMode();
         $prefs["mail_signature"] = $mailOptions->getSignature();
-        $prefs["mail_linebreak"] = $mailOptions->getLinebreak();
+        $prefs["mail_linebreak"] = ilMailOptions::DEFAULT_LINE_BREAK;
```

Comment:
@fhelfer Maybe this is causing the errors Oliver reported You have to remove **all** linebreak related code.